bundles/remote_services/remote_service_admin_dfi/gtest/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ target_link_libraries(test_rsa_dfi PRIVATE
 get_property(rsa_bundle_file TARGET rsa_dfi PROPERTY BUNDLE_FILE)
 get_property(calc_bundle_file TARGET calculator PROPERTY BUNDLE_FILE)
 get_property(calculator_shell_bundle_file TARGET calculator_shell PROPERTY BUNDLE_FILE)
-get_property(discovery_configured_bundle_file TARGET rsa_discovery_configured PROPERTY BUNDLE_FILE)
+get_property(discovery_bundle_file TARGET rsa_discovery PROPERTY BUNDLE_FILE)
```

Comment:
Change of this property name is not udpated in the config, client and server.properties

Reply:
Nice find